Sharpnado.CollectionView.Droid/Renderers/CollectionViewRenderer.cs

```diff
@@ -62,7 +63,15 @@ protected override void Dispose(bool disposing)
             }
 
             _disposed = true;
+            if (!_itemDecoration.IsNullOrDisposed())
+            {
+                _recyclerView.RemoveItemDecoration(_itemDecoration);
+                //esto elimina spaceitemdecoration pero no spacingitemdecoration
```

Comment:
is this comment helpful? What is spacingItemDecoration and why is it not interesting to delete it?